test/e2e/tests/request-queuing/dapp1-switch-dapp2-send.spec.js

```diff
@@ -240,37 +222,19 @@ describe('Request Queuing Dapp 1, Switch Tx -> Dapp 2 Send Tx', function () {
           `window.ethereum.request(${switchEthereumChainRequest})`,
         );
 
-        await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
```

Comment:
Since Dapp is already connected to Test Network, we don't need to show review permission screem

Reply:
the goal of the switch call is to trigger an approval to permit a chain that is not already permitted. This check still needs to occur. Instead the switch call above to should target 0x53a